wikitext/ui/org.eclipse.mylyn.wikitext.sdk-feature/feature.xml

```diff
@@ -43,6 +43,20 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="com.google.guava"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
```

Comment:
Having 3rd party dependency included that way will make it's version to be hardcoded in the built feature.xml thus this feature will not be installable with anything but exactly this version of guava it was buitl with. This is definetely not the wanted effect.

Reply:
Ah ok, thanks for the context.  I can see why that would be an issue.  I suppose that is also an issue with the jsoup library that's included already (e.g. that version of jsoup is forced on a consumer of this feature)?

Reply:
Isn't the feature already not installable in practice because there doesn't exist any p2 repository that provides a compatible version of Guava?